### ISSUE TITLE
Optimise HDD removal; unable to remove HDD

### ISSUE THREAD

**Issue by onlyjob** [lizardfs/lizardfs#364]
Marking HDD (i.e. mount point) for removal in `mfshdd.cfg` does nothing if chunkserver is the only one with a certain label.
In my case there is only one chunkserver labelled `poolA` - it has 10+ HDDs but nothing happens when I mark HDD for removal. I expect data from the HDD marked for removal to be distributed among local HDDs when there are no other chunkservers to send data to.
On chunkservers with many HDDs sending at least some data from HDD marked for removal to other local HDDs will be more effective than sending all data to other chunkservers.

Using local HDDs may prevent data from being moved twice -- first time when it is sent away from chunkserver and second time when data is sent back after triggering rebalancing threshold.

This bug is related to #280.

**Comment by 4Dolio** [comment]
This is a situation where few alternate rebalance methods are available.  The only other solution would be to devise a command or sctipt to manually copy the marked disks chunk contents evenly to other disks and then rescan.

**Comment by onlyjob** [comment]
It shouldn't be too hard to develop chunkserver's local asset rebalancing routine. Why manual copying (or with script) is "_the only other solution_"? Surely chunkserver code could be improved.

**Comment by 4Dolio** [comment]
I agree.. im not sure my coding skills are sufficient to fix the code properly, thus i resort to scripts for now...
